metadata-ingestion/docs/sources/neo4j/neo4j.md

```diff
@@ -0,0 +1,146 @@
+## Integration Details
+
+<!-- Plain-language description of what this integration is meant to do.  -->
+<!-- Include details about where metadata is extracted from (ie. logs, source API, manifest, etc.)   -->
+Neo4j metadata will be ingested into DataHub using 
+`CALL apoc.meta.schema() YIELD value UNWIND keys(value) AS key RETURN key, value[key] AS value;`  
+The data that is returned will be parsed 
+and will be displayed as Nodes and Relationships in DataHub.  Each object will be tagged with describing what kind of DataHub
+object it is.  The defaults are 'Node' and 'Relationship'.  These tag values can be overwritten in the recipe.
+
+
+
+## Metadata Ingestion Quickstart
+
+### Prerequisites
+
+In order to ingest metadata from Neo4j, you will need:
+
+* Neo4j instance with APOC installed
+
+
+### Install the Plugin(s)
```

Comment:
the section has been removed

Reply:
Can you confirm ? I still see it. We can completely remove the section `### Install the Plugin(s)` and all recipe details.
Let's only keep `### Prerequisites` and section mentioning `### Sample data that is returned from Neo4j.  This is the data that is parsed and used to create Nodes, Relationships.`

Reply:
I also removed the sample data section because it was incorrect.  I was originally using that sample data, but i ended up changing how i was doing thing.